store/src/main/java/dev/keva/store/impl/OffHeapDatabaseImpl.java

```diff
@@ -74,6 +74,16 @@ public void flush() {
         }
     }
 
+    @Override
+    public void close() {
+        lock.lock();
+        try {
+            chronicleMap.close();
```

Comment:
Hi @the123saurav, please consider this case https://github.com/OpenHFT/Chronicle-Map/blob/ea/docs/CM_Tutorial.adoc#close-chroniclemap

> If you call close() too early before you have finished working with the map, this can cause your JVM to crash. Close MUST be the last thing that you do with the map.

So, although it is recommended that you close() when you have finished with the map, it is not something that you must do; it’s just something that we recommend you should do.

Reply:
Hmm good point. But does that stop us from calling this? 
It will be called from 2 places:
- During JVM shutdown: In this case we are guranteed to not get any calls. 
- User initiates shutdown in embedded mode. In this case we can just add to our doc, that this should be the last thing done to KevaServer.

I am also okay to remove this from shutdown procedure.
Kindly let me know.

Reply:
Sure @the123saurav, I think we can add that warning to our doc